README.md

```diff
@@ -487,3 +487,39 @@ USER_WIDGETS=ConnectTerminalWidget;ServerWidget
 Use the file `TemplateUserWidget.py` for more practical information and to start your own widgets!
 
 
+## Using LabGUI Core Widgets ##
+
+Built into LabGUI, there are a set of core widgets in which it bases it's operation.
```

Comment:
```suggestion
Built into LabGUI, there are a set of core widgets for basic operations.
```

Reply:
You can add all commits as one if you choose "Add suggestion to batch"

Reply:
That selection is not enabled for me. My only issue with this one is that the core widgets are not designed for basic operation, but rather it forms a basis for all operations. Thoughts on how to reword in order to reflect that?